app/__init__.py

```diff
@@ -8,10 +9,77 @@
 
 
 def create_app(config_name):
+    from app.models import BucketList
+
     app = FlaskAPI(__name__, instance_relative_config=True)
     app.config.from_object(app_config[config_name])
     app.config.from_pyfile('config.py')
     app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
     db.init_app(app)
 
+    @app.route('/bucketlists/', methods=['POST', 'GET'])
+    def bucketlists():
+        if request.method == 'POST':
+            name = str(request.data.get('name', ''))
+            if name:
```

Comment:
The method was adjusted when the function was moved to `views.py` which has incorporated `MethodView`